src/providers/azure.rs

```diff
@@ -1,9 +1,46 @@
+//! Azure backend for tsunami.
+//!
+//! The primary `impl Launcher` type is [`AzureLauncher`].
+//! It internally uses the lower-level, region-specific [`AzureRegion`].
+//! Both these types use [`Setup`] as their descriptor type.
+//!
+//! Azure does not support Spot or Defined Duration instances.
+//! As a result, if your tsunami crashes (i.e., exits without calling `drop()` on [`AzureLauncher`], you must manually terminate your instances
+//! to avoid extra costs.
+//! The easiest way to do this is to delete resource groups beginning with `tsunami_`:
+//! `az group delete --name <name> --yes`.
```

Comment:
This should probably tell the user how they find resource groups beginning with `tsunami_` first. The text seems to imply that's what it does, but it is not.